<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>name_conflict_gui</class>
 <widget class="QWidget" name="name_conflict_gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Name Conflict</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Pin uses a name or label already in use.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>What do you want to do?</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="Ignore">
         <property name="text">
          <string>Keep Original</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="OverWrite">
         <property name="text">
          <string>Replace</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="Rename">
         <property name="text">
          <string>Rename</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="NewValue" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>New Label</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="NewLabel"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>New Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="NewName"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="Apply">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
